### DISCUSSION TITLE
lighthouse deck only for  lighthouse station v1 not for  lighthouse station v2

### DISCUSSION THREAD

**Discussion by ncnynl** [bitcraze/discussions#1678] [Q&A]
Recently, a customer had a problem. The deck of the lighthouse could not receive the lighthouse station v2 signal. But it could receive the lighthouse station v1 signal.

The customer flashed the new drone firmware 2024.10 version firmware, and later flashed the 2023, 2021 and other versions of the drone firmware. None of them could receive the v1 signal normally. He used the official virtual machine.

The customer once flashed the deck in reverse, and I don’t know how much impact this has.

Current status: the v2 base station is working normally, the drone is also working normally, and the lighthouse deck can only receive the v1 base station, not the v2 base station signal.

Can friends give some suggestions on how to test or restore normal use?

**Comment by tobbeanton** [comment]
My first thought. Have the BS channels been set as described [here](https://www.bitcraze.io/documentation/tutorials/getting-started-with-lighthouse/#configure-the-base-stations-channel-mode-)?

**Comment by ncnynl** [comment]
At present, we can get the signal normally with a new lighthouse deck in the same environment and base station configuration. At present, only this deck has this problem. Others are normal.

I found that if the lightouse deck is flashed with firmware for the first time, and different firmware is flashed later, it is ignored. It cannot flash different firmwares with the drone. I also want to know if there is any way to flash the lighthouse firmware separately to test whether the problem is caused by the firmware not being able to be updated.

At present, the signal of v1base is received, which should indicate that the lighthouse deck can still work normally. It may be that the signal of v2 base is not processed. Is the firmware version too low when flashing for the first time? But the firmware flashed later did not overwrite the previous firmware, which caused this problem.

**Reply by ataffanel** [reply]
For the lighthouse deck to be updated it is important to use the latest version of the Crazyflie client to flash the firmware from a release. The release will contain both Crazyflie and lighthouse firmware that matches and work well together.

The deck has to be installed on the Crazyflie when flashing the firmware as well. Do you use the latest client and can you see that the client is flashing the Lighthouse firmware as part of the update process? What operating system and python version are you using (might be relevant to reproduce problems)?
